### PR TITLE
Added FAQ entry on using Linux via remote desktop solution for windows users.

### PR DESCRIPTION
**Pull request by kunalghosh**

### CODE REVIEW

File: FAQ.md [hunk 1]
```diff
@@ -31,3 +31,18 @@ install.packages("tinytex")
 tinytex::install_tinytex()
 ```
 
+## Windows user and have a lot of problems getting packages to install ?
+
+> ⚠️ Note that **Windows is not officially supported in the course** but we will try our best to help you get your setup (in Windows) working. This is because of two main reasons:
+> 1. `Stan`, the probabilistic programming language which we will later on use in the course, doesn't have good support for Windows.
```
Comment (avehtari) [inline FAQ.md]: It should be said that "`Stan`, the probabilistic programming language requires C++ compiler toolchain which is not available by default in Windows (blame Microsoft)."

**Comment by kunalghosh** [comment]
Updated the FAQ with instructions directing to `RStan` setup page which has windows installation instructions. Also added a note about the C++ compiler tool-chain.

File: FAQ.md (hunk 1 above)
Comment (kunalghosh) [reply FAQ.md]: Just updated the FAQ. The new version should cover most of the points we discussed here.

**Comment by MansMeg** [comment]
Alright. Maybe do a PR. @avehtari ? You have any further suggestions?

**Pull request merged**